parachain/pallets/control/src/mock.rs

```diff
@@ -146,7 +149,45 @@ impl pallet_xcm_origin::Config for Test {
 }
 
 parameter_types! {
+	pub const HeapSize: u32 = 32 * 1024;
+	pub const MaxStale: u32 = 32;
+	pub static ServiceWeight: Option<Weight> = Some(Weight::from_parts(100, 100));
+}
+
+impl pallet_message_queue::Config for Test {
```

Comment:
Since we no longer need to test voucher/refund stuff, do we really still need to use a real MessageQueue in the tests?

Reply:
It's required for the new tests added in https://github.com/Snowfork/snowbridge/commit/9037a8ce0bcb74a2b231d07c67f5cdb9a8350098